content/kubernetes/security/ldap.md

```diff
@@ -0,0 +1,166 @@
+---
+Title: Enable LDAP authentication
+linkTitle: Enable LDAP
+description: Enable LDAP authentication for Redis Enterprise for Kubernetes. 
+weight: 95
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [ 
+  /kubernetes/security/ldap.md,
+  /kubernetes/security/ldap/,
+]
+---
+
+## LDAP support for Redis Enterprise Software
+
+Redis Enterprise Software supports LDAP authentication and authorization through [role-based access controls]({{<relref "/rs/security/access-control/">}}) (RBAC). You can map LDAP groups to [Redis Enterprise roles]({{<relref "/rs/security/access-control/create-roles.md">}}) to control access to your database and the [admin console]({{<relref "/rs/security/admin-console-security/">}}). For more details on how LDAP works with Redis Enterprise, see [LDAP authentication]({{<relref "/rs/security/access-control/ldap/">}}).
+
+Redis Enterprise for Kubernetes supports enabling and configuring LDAP authentication using the `RedisEnterpriseCluster` (REC) custom resource. Currently, the Redis Enterprise cluster (REC) only supports configuration related to the LDAP server, such as server addresses, connection details, credentials, and query configuration.
+
+To [map LDAP groups to Redis Enterprise access control roles]({{<relref "/rs/security/access-control/ldap/enable-role-based-ldap.md">}}), you'll need to use the Redis Enterprise [API]({{<relref "/rs/references/rest-api/requests/ldap_mappings/">}}) or [admin console]({{<relref "/rs/security/access-control/ldap/enable-role-based-ldap.md">}}).
+
+## Enable LDAP 
+
+To enable LDAP for your REC, use the `.spec.ldap` field in the `RedisEnterpriseCluster` custom resource.
+
+The following `RedisEnterpriseCluster` example resource enables a basic LDAP configuration:
+
+```yaml
+apiVersion: app.redislabs.com/v1
+kind: RedisEnterpriseCluster
+metadata:
+  name: rec
+spec:
+  nodes: 3
+  ldap:
+    protocol: LDAP
+    servers:
+    - host: openldap.openldap.svc
+      port: 389
+    bindCredentialsSecretName: ldap-bind-credentials
+    cacheTTLSeconds: 600
+    enabledForControlPlane: true
+    enabledForDataPlane: true
+    authenticationQuery:
+      template: cn=%u,ou=default,dc=example,dc=org
+    authorizationQuery:
+      attribute: memberOf
+```
+
+Refer to the `RedisEnterpriseCluster` [API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md#ldapspec) for full details on the available fields.
+
+### Bind credentials
+
+For LDAP servers that require authentication for client queries, store the bind credentials in a secret and reference them in the `RedisEnterpriseCluster` custom resource.
+
+1. Create a secret to store the bind credentials.
+    
+    ```sh
+    kubectl -n <my-rec-namespace> create secret generic ldap-bind-credentials \
```

Comment:
- `<rec-namespace>` instead of `<my-rec-namespace>` to avoid example terminology.
- `ldap-bind-credentials` is just an example secret name, we shouldn't incorrectly imply that users must use this name.
```suggestion
    kubectl -n <rec-namespace> create secret generic <secret-name> \
```